bindings/rabbitmq/rabbitmq.go

```diff
@@ -55,42 +61,82 @@ func (r *RabbitMQ) Init(metadata bindings.Metadata) error {
 
 	r.connection = conn
 	r.channel = ch
+
+	q, err := r.declareQueue()
+	if err != nil {
+		return err
+	}
+
+	r.queue = q
+
 	return nil
 }
 
 func (r *RabbitMQ) Write(req *bindings.WriteRequest) error {
-	err := r.channel.Publish("", r.metadata.QueueName, false, false, amqp.Publishing{
-		ContentType: "text/plain",
-		Body:        req.Data,
-	})
+	pub := amqp.Publishing{
+		DeliveryMode: amqp.Persistent,
+		ContentType:  "text/plain",
+		Body:         req.Data,
+	}
+
+	ttl, ok, err := bindings.TryGetTTL(req.Metadata)
 	if err != nil {
 		return err
 	}
+
+	if ok {
```

Comment:
Could you add a comment that default ttl from binding metadata has been set in declareQueue method and here its only overriddenif its present in request metadata.